src/daemon/http/server.rs

```diff
@@ -508,6 +515,49 @@ pub async fn metrics(req: Request) -> RoutingResult {
                 res.push_str(&format!("krill_cas_bgp_announcements_valid{{ca=\"{}\"}} {}\n", ca, nr));
             }
 
+            res.push('\n');
+            res.push_str("# HELP krill_ca_parent_success status of last ca-parent connection (0=issue, 1=success)\n");
+            res.push_str("# TYPE krill_ca_parent_success gauge\n");
```

Comment:
Should this be a counter rather than a gauge as a counter is for values that "can only increase or be reset to zero on restart" while a gauge is for a value that "can arbitrarily go up and down" ? (based on https://prometheus.io/docs/concepts/metric_types/)

Reply:
I meant to also say that I can imagine scenarios where time doesn't just increase as expected so maybe that's another reason for a gauge?

Reply:
Typically you would not reset a counter to zero except on program restart. 

If you want this to be a counter you could count the number of successful events and monitor how long it did not increase

Reply:
I suppose this is opinion and not gospel, but still well phrased:
https://www.robustperception.io/are-increasing-timestamps-counters-or-gauges

In short: use gauge for timestamp. Time can even go backwards due to NTP.

Also, we have been using gauge for existing timestamp metrics (krill_server_start).

Reply:
> Time can even go backwards due to NTP.

Yup, I alluded to that above. Gauge it is then.